packages/perseus-editor/src/widgets/table-editor.tsx

```diff
@@ -93,7 +93,7 @@ class TableEditor extends React.Component<Props> {
     };
 
     render(): React.ReactNode {
-        const tableProps = _.pick(
+        const tableProps: any = _.pick(
```

Comment:
I don't think this inferred type was correct and it was causing an error.

Reply:
tbh, I'd much rather we remove `tableProps` entirely and pass the props to the `<Table>` component on line 156 explicitly.